### PR TITLE
Create database indices on recently added fields

### PR DESCRIPTION
**Pull request by jgiannuzzi**
I forgot to ensure the new `runs.row_num` and `metrics.iter` fields were indexed. This update should increase query speed in a few operations.

### CODE REVIEW

File: pkg/database/database.go [hunk 1]
```diff
@@ -311,6 +311,23 @@ func ConnectDB(dsn string, slowThreshold time.Duration, poolMax int, reset bool,
...
+				log.Info("Migrating database to fasttrack schema ed364de02645")
+				if err := DB.Transaction(func(tx *gorm.DB) error {
+					if err := tx.Migrator().CreateIndex(&Run{}, "RowNum"); err != nil {
+						return err
+					}
+					if err := tx.Migrator().CreateIndex(&Metric{}, "Iter"); err != nil {
+						return err
+					}
+					return tx.Model(&SchemaVersion{}).
+						Where("1 = 1").
```
Comment (dannyfriar) [inline pkg/database/database.go:324]: Why is this needed?

**Review by dannyfriar — APPROVED** [review]

File: pkg/database/database.go (hunk 1 above)
Comment (jgiannuzzi) [reply pkg/database/database.go:324]: This is a quirky way to tell `gorm` to one row we have in that table. See the [docs](https://gorm.io/docs/update.html#Block-Global-Updates) for more details.
